k8s/upgrade/src/preflight_validations.rs

```diff
@@ -118,7 +121,7 @@ pub async fn rebuild_in_progress_validation(
 }
 
 /// Check for rebuild in progress.
-pub async fn rebuild_in_progress(client: &upgrade_lib::RestClient) -> Result<bool, error::Error> {
+pub async fn rebuild_in_progress(client: &upgrade_lib::RestClient) -> Result<bool, Error> {
```

Comment:
nit: `rebuild_in_progress` --> `check_rebuild_in_progress`. I know its in doc comment, but the method name should also reflect that?